templates/.github/workflows/go-test.yml

```diff
@@ -7,7 +7,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ "ubuntu", "windows", "macos" ]
-        go: [ "1.16.x", "1.17.x" ]
+        go: ${{ config.go.versions }}
```

Comment:
How is this being replaced? Wouldn't this need the templating PR?

Reply:
Again, forgot to stage my latest changes - https://github.com/protocol/.github/pull/280/commits/9bd61de7c19ceb2be6e0f13ea180c408330c280a

You've spotted correctly that it should use the template engine syntax - `${{{ config.go.versions }}}`.

And yes - this PR requires #262. I mentioned it in the `Prerequisites` section of the description.